teams/4/page-1.css

```diff
@@ -925,6 +929,210 @@ ul li:not(:first-of-type):not(.company__logo) {
   }
 }
 
+/* sliderSection (Nazar) */
+
+.sliderSection {
+  padding: 96px 112px;
+  padding-bottom: 186px;
+}
+
+.sliderHeading {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 64px;
+  margin-right: 15px;
+}
+
+.sliderHeadingText {
+  margin-bottom: 20px;
+  margin-right: 20px;
+}
+
+.sliderHeadingBtn,
+.customBtn {
+  border-radius: 8px;
+  padding: 12px 20px;
+  background-color: var(--color-primary-600);
+  border: 0;
+  cursor: pointer;
+  color: var(--color-white);
+}
+
+.sliderList {
+  display: flex;
+}
+
+.slick-track {
+  display: flex !important;
```

Comment:
the use of important is extremely forbidden, please asap remove it all over

Reply:
@iamcucusa  here I'm overriding the slick-slider plugin's default styles. Without !important it is impossible to override styles.